Review, format and adjust configuration.rb

markuspg opened the pull request:
This gives `configuration.rb` the exactly same treatment as `base.rb`. As only functional change from the call of `@logger.info` the safe navigation operator has been dropped, since `@logger` should always reference a valid logger instance (and other invocations of `@logger` also don't rely on the safe navigation operator).

markuspg committed:
Fix RuboCop offenses for configuration.rb tests

markuspg committed:
Increase test coverage of configuration.rb

markuspg committed:
Fix RuboCop offenses for configuration.rb

mrkita commented:
It would be nice to see the functional change in it's own commit... am I just missing it?

markuspg committed:
Drop safe navigation operator from @logger

@logger should always contain a valid logger instance. Therefore this
ocurrence of the safe navigation operator is superfluous.
